disnake/embeds.py

```diff
@@ -371,6 +383,14 @@ def timestamp(self, value: Optional[datetime.datetime]) -> None:
                 f"Expected datetime.datetime or None received {type(value).__name__} instead"
             )
 
+    @property
+    def flags(self) -> Optional[EmbedFlags]:
+        """Optional[:class:`EmbedFlags`]: Returns the embed's flags.
+
+        .. versionadded:: 2.11
+        """
+        return EmbedFlags._from_value(self._flags)
```

Comment:
This no longer returns `None`
```suggestion
    @property
    def flags(self) -> EmbedFlags:
        """:class:`EmbedFlags`: Returns the embed's flags.

        .. versionadded:: 2.11
        """
        return EmbedFlags._from_value(self._flags)
```